### PR TITLE
chore(docs): Add map-deck initial docs

### PR DESCRIPTION
**Pull request by GordonSmith**
Sort @hpcc-js packages in hpccIndex.ts
Catch runtime "SET" publish properties
Fix initial sample height is too small
Add a default max-height for samples

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,4 @@ tmp*.json
 *.xml
 *.zip
 *.tsbuildinfo
+/website/api
```
Comment (jbrundage) [inline .gitignore]: What will this section be used for?

File: packages/map-deck/docs/CircleLines.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+            "type": "reference",
+            "name": "Common",
+            "id": 10
+        }
+    ],
+    "folder": "packages/map-deck"
+}
+-->
+
+Similar to @hpcc-js/map -> CirlceLines - this widget is a 3D map with to/from points and connecting lines.
```
Comment (jbrundage) [inline packages/map-deck/docs/CircleLines.md]: Should `@hpcc-js/map -> CirlceLines` be a link?

File: packages/map-deck/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# @hpcc-js/map-deck
+
+This package is @hpcc-js wrapper around [Deck.GL](https://deck.gl/).
```
Comment (jbrundage) [inline packages/map-deck/docs/index.md]: Shouldn't this be "is an @hpcc-js wrapper"?

File: .gitignore (hunk 1 above)
Comment (GordonSmith) [reply .gitignore]: Its where the generated typedoc API ends up

**Pull request merged**